templates/e-commerce/checkout.amp.json

```diff
@@ -72,19 +73,22 @@
           "name": "delivery-method",
           "id": "standard",
           "label": "Standard (arrives in 2-3 days) - FREE",
-          "checked": "checked"
+          "checked": "checked",
+          "on": "change:AMP.setState({cart: {total: 723, shipping: 0}})"
         },
         {
           "name": "delivery-method",
           "id": "express",
-          "label": "Express (arrives next day) - £30"
+          "label": "Express (arrives next day) - £30",
+          "on": "change:AMP.setState({cart: {total: 753, shipping: 30}})"
         }
       ]
     }
   },
   "cart": {
     "header": {
       "responsive-layout-classes": "xs-hide sm-hide"
-    }
+    },
+    "dynamic": true
```

Comment:
Whether the price should update using amp-bind (cart page) or be static (checkout page).